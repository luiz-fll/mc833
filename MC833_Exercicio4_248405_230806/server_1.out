remoto: 127.0.0.1:6001
[SERVIDOR] Escutando em 127.0.0.1:6001
127.0.0.1:36044
GET / HTTP/1.0
Host: teste

127.0.0.1:36056
GET / HTTP/1.0
Host: teste

127.0.0.1:36072
GET / HTTP/1.0
Host: teste

127.0.0.1:36078
GET / HTTP/1.0
Host: teste

127.0.0.1:36094
GET / HTTP/1.0
Host: teste

127.0.0.1:36098
GET / HTTP/1.0
Host: teste

127.0.0.1:36100
GET / HTTP/1.0
Host: teste

127.0.0.1:36108
GET / HTTP/1.0
Host: teste

127.0.0.1:36116
GET / HTTP/1.0
Host: teste

