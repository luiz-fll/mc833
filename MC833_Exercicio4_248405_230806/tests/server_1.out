remoto: 127.0.0.1:60001
[SERVIDOR] Escutando em 127.0.0.1:60001
127.0.0.1:54620
GET / HTTP/1.0
Host: teste

127.0.0.1:54622
GET / HTTP/1.0
Host: teste

127.0.0.1:54630
GET / HTTP/1.0
Host: teste

127.0.0.1:54636
GET / HTTP/1.0
Host: teste

127.0.0.1:54644
GET / HTTP/1.0
Host: teste

127.0.0.1:54648
GET / HTTP/1.0
Host: teste

127.0.0.1:54650
GET / HTTP/1.0
Host: teste

127.0.0.1:54654
GET / HTTP/1.0
Host: teste

127.0.0.1:54662
GET / HTTP/1.0
Host: teste

